### PR TITLE
Update all.json

### PR DESCRIPTION
**Pull request by ccris02**
|	Site	|	Urlscan	|
|	----------------------------------------------	|	----------------------------------------------	|
|	babyswaps.to	|	https://urlscan.io/result/a07280a4-4eba-4202-b3c2-295b994bbb53/	|
|	biaswap.org	|	https://urlscan.io/result/0e6e315d-6aee-4327-9d4d-9bbf97c11369/	|
|	defispool.net	|	https://urlscan.io/result/c135f0b9-7e12-47dc-a9b9-9c3c56f731ec/	|
|	factor-swap.com	|	https://urlscan.io/result/389a109b-56a9-4bd5-b751-48368b38a1bf/	|
|	gdtokenpocket.com	|	https://urlscan.io/result/b38fddc0-4157-4c98-8f50-9dbd7a1b288e/	|
|	im1token.com	|	https://urlscan.io/result/98f4f467-91ec-440f-959c-79e387cc2272/	|
|	imiltoken.com	|	https://urlscan.io/result/2f86594e-fca5-4d9d-9fdc-8cba060d30b1/	|
|	im-token.pub	|	https://urlscan.io/result/ff6772b5-cdfd-4a8d-9451-31b3416e2997/	|
|	landsync.live	|	https://urlscan.io/result/fc9cf9e1-e9eb-4f39-be1f-75f1b3948eff/	|
|	limtoken.cc	|	https://urlscan.io/result/c358ac84-d193-48c0-afba-491c46242467/	|
|	lmtokenm.im	|	https://urlscan.io/result/cf89786a-ad7f-4154-bcab-9093d7472919/	|
|	syncyourapp.info	|	https://urlscan.io/result/7c4d7f07-e045-41fa-a8fd-064a288cff79/	|
|	syncyourapp.online	|	https://urlscan.io/result/89a5531e-91f9-45f7-bbcd-d46b9197ede8/	|
|	tokenporcket.cc	|	https://urlscan.io/result/9ecaed20-9531-485f-ba11-e749a3fc286f/	|
|	trusttoken.info	|	https://urlscan.io/result/9a6edb5f-164d-4725-be38-e2a581818652/	|
|	trusttoken.top	|	https://urlscan.io/result/bd18d864-6210-4c9a-b753-2a564d676e6a/	|
|	walletbridgedapp.net	|	https://urlscan.io/result/2fd01ae5-0635-4204-88ba-8b448127af2f/	|
|	walletsencrypt.net	|	https://urlscan.io/result/c00ec62b-cc71-41fe-94ef-86a1d9b398c1/	|
|	web3nodesync.com	|	https://urlscan.io/result/6d942f87-75e5-4ec6-a415-32b35b4cf565/	|

### CODE REVIEW

**Comment by ical10** [comment]
you have some dupes:
```
    + Array [
    +   "dappsconnectchain.com",
    +   "synchconnect.tk",
    + ]
```

**Review by jacogr — APPROVED** [review]

File: all.json [hunk 1]
```diff
@@ -2,6 +2,27 @@
 	"allow": [],
 	"deny": [
 		"07e96d43-381a-46a3-9c16-6daf97213efc.co",
+		"babyswaps.to",
+		"biaswap.org",
+		"dappsconnectchain.com",
```
Comment (jacogr) [inline all.json:7]:
```suggestion
```

https://github.com/polkadot-js/phishing/runs/6071331993?check_suite_focus=true#step:3:34

File: all.json [hunk 2]
```diff
@@ -2,6 +2,27 @@
...
+		"defispool.net",
+		"factor-swap.com",
+		"gdtokenpocket.com",
+		"im1token.com",
+		"imiltoken.com",
+		"im-token.pub",
+		"landsync.live",
+		"limtoken.cc",
+		"lmtokenm.im",
+		"synchconnect.tk",
```
Comment (jacogr) [inline all.json:17]:
```suggestion
```

https://github.com/polkadot-js/phishing/runs/6071331993?check_suite_focus=true#step:3:35

**Review by jacogr — CHANGES_REQUESTED** [review]
Removed dupes

**Review by ical10 — APPROVED** [review]

**Comment by ical10** [comment]
All good now? @jacogr

**Review by frankywild — APPROVED** [review]

**Pull request merged**